Auth0/SafariWebAuth.swift

```diff
@@ -153,6 +158,7 @@ class SafariWebAuth: WebAuth {
             entries["nonce"] = self.nonce
         }
         self.parameters.forEach { entries[$0] = $1 }
+        self.parameters = [:]
```

Comment:
This is intended since `WebAuth` should not be reusable in that way since its the same instance so I'd not make this hidden side-effect here. Better to properly handle this in Lock